CHANGELOG.md

```diff
@@ -28,6 +28,11 @@
   }
   ```
 
+- Added retry implementation when making an API call which retries when:
```

Comment:
This changelog for retry got moved above, so this can be removed

Reply:
ok done, and also moved the line about `exportDocumentAsStream` above the fixes section